bioblend/_tests/TestGalaxyJobs.py

```diff
@@ -75,6 +77,27 @@ def test_get_jobs(self):
         self.assertEqual(len(jobs), 0)
         jobs = self.gi.jobs.get_jobs(date_range_min=datetime.today().strftime('%Y-%m-%d'), history_id=self.history_id)
         self.assertEqual(len(jobs), 3)
+        job_id_lim = jobs[1]['id']
```

Comment:
These tests will fail on older versions of Galaxy. Maybe the best solution would be to move these additions to a new method `test_get_jobs_with_filtering` and add a `@test_util.skip_unless_galaxy('release_21.05')` decorator?